lxd/storage_utils.go

```diff
@@ -35,7 +36,7 @@ var MountOptions = map[string]mountOptions{
 	"diratime":      {false, unix.MS_NODIRATIME},
 	"dirsync":       {true, unix.MS_DIRSYNC},
 	"exec":          {false, unix.MS_NOEXEC},
-	"lazytime":      {true, MS_LAZYTIME},
+	"lazytime":      {true, driver.MS_LAZYTIME},
```

Comment:
This is appears to be defined in the `unix` package nowadays so we can switch that to unix.MS_LAZYTIME and drop the special logic in cgo.

Reply:
This has been fixed upstream.